From: Alexander Larsson <alexl@redhat.com>
Date: Fri, 20 May 2016 10:59:59 +0200
Subject: tests: Fix build with old glib

test-checksum.c was using g_autofree without including libglnx.h which
has the backport for that.

Closes: #304
Approved by: cgwalters
Origin: upstream, 2016.6, commit:86ac8062f5311fe94ef4096f4e136ce993d5e9b2
---
 tests/test-checksum.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test-checksum.c b/tests/test-checksum.c
index 25d3c37..e7fd7be 100644
--- a/tests/test-checksum.c
+++ b/tests/test-checksum.c
@@ -20,6 +20,7 @@
 
 #include "config.h"
 
+#include "libglnx.h"
 #include "libgsystem.h"
 #include <glib.h>
 #include <stdlib.h>
